deposit/src/main/java/edu/unc/lib/deposit/transfer/TransferBinariesToStorageJob.java

```diff
@@ -91,6 +92,15 @@ public void runJob() {
         Model model = getReadOnlyModel();
         Bag depositBag = model.getBag(getDepositPID().getRepositoryPath());
 
+        // Count how many objects are being deposited
+        int i = 0;
+        ResIterator subjectIterator = model.listSubjects();
```

Comment:
I don't think the click counts will line up with the total anymore, since clicks are still counting number of files while the total is the number of resources. I believe you'll need to move the `addClicks` call into `transferBinaries` since that iterates over each resource in the model.